src/state/sagas/datasets/QueryDatasetTwingraph/QueryDatasetTwingraph.js

```diff
@@ -44,8 +44,16 @@ function* runDatasetTwingraphQuery(action, query, attemptsNumber = 0) {
 function* startAllDatasetTwingraphQueries(action) {
   const { workspace } = action.payload;
   const queries = workspace?.webApp?.options?.datasetManager?.queries ?? [];
+  const graph = workspace?.webApp?.options?.datasetManager?.graphIndicators;
+  const categories = workspace?.webApp?.options?.datasetManager?.categories;
+
   yield all(
     queries.map((query, index) => {
+      if (
+        !categories.some((cat) => cat?.kpis.some((kpi) => kpi?.queryId === query.id)) &&
+        !graph.some((g) => g?.queryId === query.id)
```

Comment:
Could we rename `g` to something like `kpi`, `indicator` or `graphIndicator`?
Also, `graph` is an array, maybe `graphIndicators` would be a better name for it